packages/persist-state/src/lib/persist-state.ts

```diff
@@ -59,7 +61,8 @@ export function persistState<S extends Store>(store: S, options: Options<S>) {
   return {
     initialized$: initialized.asObservable(),
     unsubscribe() {
-      subscription.unsubscribe();
+      saveToStorageSubscription.unsubscribe();
```

Comment:
I have no preference about the order from which to unsubscribe, so i used the default LIFO mode: last acquired resource is released first.